packages/curve-ui-kit/src/shared/ui/DataTable.tsx

```diff
@@ -30,17 +33,20 @@ const getAlignment = <T extends TableItem>({ columnDef }: Column<T>) =>
 
 const DataCell = <T extends TableItem>({ cell }: { cell: Cell<T, unknown> }) => {
   const column = cell.column
+  const { hidden, variant, borderRight } = column.columnDef.meta ?? {}
   return (
-    !column.columnDef.meta?.hidden && (
+    !hidden && (
       <Typography
-        variant={column.columnDef.meta?.variant ?? 'tableCellMBold'}
+        variant={variant ?? 'tableCellMBold'}
         color="text.primary"
         component="td"
         sx={{
           textAlign: getAlignment(column),
           paddingInline: Spacing.sm,
           paddingBlock: Spacing.xs, // `md` removed, content should be vertically centered
           ...getExtraColumnPadding(column),
+          transition: `color ${TransitionFunction}, border-right ${TransitionFunction}`,
```

Comment:
Is this transition still necessary now that it's part of `create-typography.ts` on line 64?

Reply:
indeed, can be removed